ScriptingEngineTester/src/main/resources/zencode-tests/null-coalesce/null-call-2.zc

```diff
@@ -2,7 +2,7 @@
 #output: hello
 
 function test(): string? => null;
-function test(): string? => "hELLo";
+function test2(): string? => "hello";
 
 println(test()?.toLowerCase());
-println(test()?.toLowerCase());
+println(test2()?.toLowerCase());
```

Comment:
Is there a reason to change this test?

Reply:
Well, I doubt the test intended 2 functions with the name `test` to be present, did it?

Reply:
I understand the name change, but I wanted to double-check that toLowerCase was also properly called, hence the uppercase letters in the test ;)

Reply:
Reverted 👍🏽